### PR TITLE
Ec q regulator search

### PR DESCRIPTION
**Pull request by chrisxu3**
Adds regulator search to elliptic curves over Q, resolving #3238
Right now the code fails because 'regulator' is a column in ec_curves_extras, but I'm told David is working on getting it moved to ec_curves.

### CODE REVIEW

**Comment by roed314** [comment]
I've moved `regulator` from `ec_curves_extras` to `ec_curves` so this should work now.

File: lmfdb/elliptic_curves/templates/ec-index.html [hunk 1]
```diff
@@ -210,6 +210,15 @@ <h2> Search </h2>
     <td>
     <span class="formexample"> e.g. 2 or 4-15</span>
     </td>
+    <td>
+    {{ KNOWL('ec.q.regulator',title="Regulator") }}
+    </td>
+    <td>
+    <input type='text' name='regulator' example="8.4-9.1" size=10>
```
Comment (alexjbest) [inline lmfdb/elliptic_curves/templates/ec-index.html]: I think the HTML is a bit funny here, there should be a closing `</td>` before another opening one

**Comment by alexjbest** [comment]
The file `lmfdb/templates/.knowl-defs.html.sage-chat` should  not be commited.

File: lmfdb/elliptic_curves/templates/ec-index.html (hunk 1 above)
Comment (chrisxu3) [reply lmfdb/elliptic_curves/templates/ec-index.html]: Fixed, thanks

**Pull request merged**